lib/utils.js

```diff
@@ -67,7 +67,7 @@ var self = module.exports = {
 
 	// Escaping values:
 	// http://ircv3.net/specs/core/message-tags-3.2.html#escaping-values
-	unescapeIRC: msg => !msg || !msg.includes('\\') ?
+	unescapeIRC: msg => !msg || (msg && !msg.includes('\\')) ?
```

Comment:
The change doesn't do anything that the "`!msg`" isn't already doing, effectively no change. It should do a type check on `msg`.
```js
unescapeIRC: !msg || typeof msg !== 'string' || !msg.includes('\\') ?
```
Check out short circuiting.

Reply:
Thank you for your review. :)